packages/skeleton-react/src/components/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,179 @@
+import { describe, expect, it } from "vitest";
+import { render, waitFor } from "@testing-library/react";
+
+import { Accordion, AccordionContext } from "./Accordion";
+
+// Accordion ---
+
+describe("<Accordion>", () => {
+  it("should render the component", () => {
+    const component = render(<Accordion />);
+    expect(component).toBeTruthy();
+  });
+
+  it("should render with `multiple` prop enabled", () => {
+    const component = render(<Accordion multiple />);
+    expect(component).toBeTruthy();
+  });
+
+  it("should allow for children", () => {
+    const value = "foobar";
+    const { getByTestId } = render(<Accordion>{value}</Accordion>);
+    expect(getByTestId("accordion").innerHTML).toContain(value);
+  });
+
+  it("should allow you to set the `base` style prop", () => {
+    const tailwindClasses = "bg-red-500";
+    const { getByTestId } = render(<Accordion base={tailwindClasses} />);
+    expect(getByTestId("accordion").classList).toContain(tailwindClasses);
```

Comment:
you actually don't need the `.classList` approach to check if classes are applied you can use the `toHaveClass(tailwindClasses)` assertion

Reply:
See: https://github.com/skeletonlabs/skeleton/pull/2566#discussion_r1539983435